### PR TITLE
Create and remove workspaces

### LINKED ISSUE #64: Create workspaces via command
I was thinking that now with the masked tokens input, shouldn't we make it possible to create workspaces via the `config` command, @voidpirate?

It could be via a `--create` argument that asks for info just like `--init` does it, and also checks auth of the token.

### PR DESCRIPTION
**Pull request by netromdk**
Depends on #67 to be merged first so master can be rebased onto this one.

Fixes #64.

Changes proposed in this pull request:
- Convenience `create_logger` that respects quiet mode if defined 
- `verify_token` utility function
- AuthTestCommand uses `verify_token`
- `workspace_prompt` function used in `--init` and `WorkspaceCommand`
- `--create` and `--remove` workspaces via command

### CODE REVIEW

**Comment by netromdk** [comment]
Had an idea: By checking a token with `auth.test` it returns a response like this:
```
2018-02-06 20:44:11,061 - AuthTestCommand - DEBUG - URL: https://SPACE.slack.com/
2018-02-06 20:44:11,061 - AuthTestCommand - DEBUG - Workspace: SPACE
2018-02-06 20:44:11,061 - AuthTestCommand - DEBUG - Workspace ID: SPACE_ID
2018-02-06 20:44:11,061 - AuthTestCommand - DEBUG - User: USER
2018-02-06 20:44:11,061 - AuthTestCommand - DEBUG - User ID: USER_ID
```
So why don't we change it so that adding a new workspace only requires inputting a token, then we simply grab the name of the workspace on successful authentication instead. Small thing but I like to make things simple as can be.

File: slacker/utility.py [hunk 1]
```diff
@@ -72,3 +74,26 @@ def parse_args():
...
   return (args, cmd_args)
+
+def verify_token(token):
+  try:
+    return SlackAPI(token).post('auth.test')
+  except Exception as ex:
+    create_logger(__name__).warning(ex)
+  return None
+
+def workspace_prompt(msg = 'Input workspace token: '):
```
Comment (voidpirate) [inline slacker/utility.py]: Can we rename this to be `workspace_token_prompt` so it's a bit more informative. At first I wasn't sure what it was doing.

File: slacker/utility.py [hunk 2]
```diff
@@ -72,3 +74,26 @@ def parse_args():
 
   args = parser.parse_args(args)
   return (args, cmd_args)
+
+def verify_token(token):
```
Comment (voidpirate) [inline slacker/utility.py]: I wonder if we should convert this to be a decorator that we apply to `action` in the command, at least where the commands require a valid auth token? Although that might be a bit spamming if we are making an auth test call before running the command.

File: slacker/utility.py (hunk 1 above)
Comment (netromdk) [reply slacker/utility.py]: Sure, I can rename it.

File: slacker/utility.py (hunk 2 above)
Comment (netromdk) [reply slacker/utility.py]: Hm that would make too many requests to the API that are unnecessary. We will get an unauthed error if using an invalid token anyway.

**Pull request merged**